src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
Variables and methods should not be static
```suggestion
    private final Random random = new Random();

    public Color getRandomColor() {
```